include/irr/asset/IAssetLoader.h

```diff
@@ -91,13 +109,14 @@ class IAssetLoader : public virtual core::IReferenceCounted
 
     struct SAssetLoadParams
     {
-        SAssetLoadParams(const size_t& _decryptionKeyLen = 0u, const uint8_t* _decryptionKey = nullptr, const E_CACHING_FLAGS& _cacheFlags = ECF_CACHE_EVERYTHING)
-            : decryptionKeyLen(_decryptionKeyLen), decryptionKey(_decryptionKey), cacheFlags(_cacheFlags)
+        SAssetLoadParams(const size_t& _decryptionKeyLen = 0u, const uint8_t* _decryptionKey = nullptr, const E_CACHING_FLAGS& _cacheFlags = ECF_CACHE_EVERYTHING, const E_LOADER_PARAMETER_FLAGS _loaderFlags = ELPF_NONE)
+            : decryptionKeyLen(_decryptionKeyLen), decryptionKey(_decryptionKey), cacheFlags(_cacheFlags), loaderFlags(_loaderFlags)
         {
         }
-        size_t decryptionKeyLen;			 //!< The size of decryptionKey
-        const uint8_t* decryptionKey;		 //!< The key it used to decrypt potentially encrypted files
-        const E_CACHING_FLAGS cacheFlags;	 //!< Flags defining rules during loading process
+        size_t decryptionKeyLen;				    	//!< The size of decryptionKey
+        const uint8_t* decryptionKey;				    //!< The key it used to decrypt potentially encrypted files
+        const E_CACHING_FLAGS cacheFlags;		     	//!< Flags defining rules during loading process
+		const E_LOADER_PARAMETER_FLAGS loaderFlags;		//!< Flags having an impact on extraordinary tasks during loading process
```

Comment:
@devshgraphicsprogramming shouldn't it actually be a uint64_t, should it? I mean, you could use then || and specify more than one flag at once

Reply:
nope all good.